src/Orckestra.Composer.Website/App_Data/Razor/Composer/SingleCheckout/Steps/ReviewCart.cshtml

```diff
@@ -235,15 +235,29 @@
 
 @helper CartItemRecurringOrderProgram()
```

Comment:
Lets move this helper outside to App_Code/RecurringOrderHelpers.chtml
So later we can re-use this HTML when we will rewrite Cart Page to use vuew as well.